src/lm.jl

```diff
@@ -156,7 +156,32 @@ function coeftable(mm::LinearModel)
               ["x$i" for i = 1:size(mm.pp.X, 2)], 4)
 end
 
-predict(mm::LinearModel, newx::Matrix) =  newx * coef(mm)
+predict(mm::LinearModel, newx::AbstractMatrix) = newx * coef(mm)
+
+"""
+For linear models, specifying `interval_type` will return a 3-tuple with the
```

Comment:
Please add the signature first (with a four-space indent and a line break after it).